src/Application/UI/Presenter.php

```diff
@@ -230,7 +234,17 @@ public function run(Application\Request $request): Application\Response
 
 			// SIGNAL HANDLING
 			// calls $this->handle<Signal>()
-			$this->processSignal();
+			$signalException = null;
+			try {
+				$this->processSignal();
+			} catch (Throwable $signalException) {
```

Comment:
I think the catch can be omitted

Reply:
It is used to assign Throwable to $signalException

Reply:
Remove it too. Just try ... finally